src/scales/scale.linear.js

```diff
@@ -12,18 +12,19 @@ const defaultConfig = {
 
 class LinearScale extends LinearScaleBase {
 	determineDataLimits() {
-		var me = this;
-		var DEFAULT_MIN = 0;
-		var DEFAULT_MAX = 1;
-		var minmax = me._getMinMax(true);
-		var min = minmax.min;
-		var max = minmax.max;
+		const me = this;
+		const options = me.options;
+		const DEFAULT_MIN = options.suggestedMin || 0;
+		const DEFAULT_MAX = options.suggestedMax || 1;
```

Comment:
is it possible for suggestedMax to be 0 (eg when min is < 0)?

Reply:
Yes, that's a good point. This should use the helper that checks for undefined